DxDispatch/src/model/JsonParsers.cpp

```diff
@@ -1015,6 +1017,38 @@ std::vector<std::byte> GenerateInitialValuesFromSequence(DML_TENSOR_DATA_TYPE da
     }
 }
 
+std::pair<std::vector<std::byte>, DML_TENSOR_DATA_TYPE> GenerateInitialValuesFromFile(const rapidjson::Value& object)
+{
+    auto fileName = ParseStringField(object, "sourcePath");
+    auto wFileName = std::wstring_convert<std::codecvt_utf8<wchar_t>>().from_bytes(fileName);
+
+    std::ifstream file(wFileName.c_str(), std::ifstream::ate | std::ifstream::binary);
```

Comment:
Yeah, evidently Windows VS `fstream` has a `wchar_t` overload (because all OS strings on Windows since NT were UTF-16), but not on Ubuntu WSL. I'll have to think about how to make this work across both platforms so that paths with non-ASCII names work correctly (perhaps std::filesystem::path can help...).